src/main/java/uk/gov/companieshouse/model/validator/AppealReasonValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.model.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.exception.AppealReasonException;
+import uk.gov.companieshouse.model.Reason;
+
+@Component
+public class AppealReasonValidator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AppealReasonValidator.class);
+
+    public void validate(Reason reason) throws AppealReasonException {
+        if(reason.getReasonType() != null){
+            LOGGER.info("Correctly posted {} appeal reason", reason.getReasonType());
```

Comment:
This will create noise in the logs. It doesn't identify which user or appeal this is for. A single log line in the controller that the Appeal has been validated successfully would reduce the number of logs if we add more validation in the future.